src/functions-and-arrays.js

```diff
@@ -1,41 +1,81 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(x, y) {
+  if (x > y) {
+    return x
+  } else if (x < y) {
+    return y
+  } else x
+}
+
```

Comment:
Te faltaba retornar el último caso

```suggestion
function maxOfTwoNumbers(x, y) {
  if (x > y) {
    return x
  } else if (x < y) {
    return y
  } else {
    return x
  }
}

```